src/coreclr/scripts/antigen_run.py

```diff
@@ -213,6 +213,12 @@ def main(main_args):
         # Run tool such that issues are placed in a temp folder
         with TempDir() as temp_location:
             antigen_log = path.join(temp_location, get_antigen_filename(tag_name))
+            env_variable = {}
+            if 'arm' not in run_duration:
```

Comment:
```suggestion
            if 'x86' in coreclr_args.run_configuration or 'x64' in coreclr_args.run_configuration:
```

Reply:
i thought about this initially, but `arm` checked looked reasonable to me. I will change it back to x86/x64.

Reply:
It was mainly the `run_duration` => `run_configuration` change that is important, but I think it's better to be inclusive rather than exclusive for something like this.